### DISCUSSION TITLE
Error when attempting to customise the dashboard via local.yaml file - Unrecognized option "dashboard" under "kimai"

### DISCUSSION THREAD

**Discussion by 1amassu** [kimai/kimai#4286] [Questions & Answers] [answered]
As per the documentation (https://www.kimai.org/documentation/dashboard.html) I was trying to customize the dashboard, however when following the syntax in the documentation, I am receiving the following error:

 [ERROR] Unrecognized option "dashboard" under "kimai". Available options are "activity", "calendar", "company",
         "customer", "data_dir", "defaults", "export", "features", "invoice", "ldap", "permissions", "project",
         "quick_entry", "saml", "theme", "timesheet", "user".

I have double and triple checked the documentation. I am on the latest version (2.0.3.3) and am currently running this in a shared environment (Cloudways is the provider)

**Comment by kevinpapst** [comment, marked as answer]
Thanks 👍 

I updated the docs to match 2.0: https://www.kimai.org/documentation/dashboard.html

**Comment by 1amassu** [comment]
Thank you! I had to re-read that same page 100 times to make sure I wasn't mistaken. My initial goal was to build a custom widget to replace "my projects" which will display 'my projects' even when no budget is allocated. Is this no longer possible? By the way I only discovered, researched, installed and configured Kimai today! Amazingly well built.

**Reply by 1amassu** [reply]
Does this help me at all? https://www.kimai.org/documentation/developers.html#extending-the-dashboard-with-widgets

**Reply by kevinpapst** [reply]
Appreciate the kind words 😄 

You can add widgets via your own plugin:
https://www.kimai.org/documentation/plugins.html

To start off, simply clone: 
https://github.com/Keleo/DemoBundle

Then you can register new wigets via EventSubscriber:
https://www.kimai.org/documentation/developers.html#extending-the-dashboard-with-widgets

I really hope the docs are up-to-date. Please let me know if something doesn't work as documented.

**Reply by 1amassu** [reply]
Appreciate the quick replies! The Docs have been an amazing help so far, I will definitely let you know if I get stuck (after I check 1000x to make sure it's not my fault 😆 )

**Reply by kevinpapst** [reply]
I am doing most by myself, so the docs might be lagging behind. 😁
